### PR TITLE
Adding codemeta.json file for standard metadata

### PR DESCRIPTION
**Pull request by vuillaut**
@garciagenrique @LukasNickel maybe you can add your author information  as  well in the file please - by pushing directly to this PR branch is fine

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "dateCreated": "2019-09-09",
+    "datePublished": "2020-05-04",
+    "dateModified": "2021-04-21",
+    "downloadUrl": "https://github.com/cta-observatory/lstmcpipe/archive/refs/tags/v0.4.1.tar.gz",
+    "issueTracker": "https://github.com/cta-observatory/lstmcpipe/issues",
+    "name": "lstmcpipe",
+    "version": "0.4.1",
+    "description": "Scripts to ease the reduction of MC data on the LST cluster at La Palma. With this package, the analysis/creation of R1/DL0/DL1/DL2/IRFs can be orchestrated.",
+    "applicationCategory": "CTA, LST",
+    "releaseNotes": "Working version with\n\n - `lstchain` [v0.6.3](https://github.com/cta-observatory/cta-lstchain/releases/tag/v0.6.3)\n -  `HiPeRTA` [v4.0.0-dev](https://gitlab.in2p3.fr/CTA-LAPP/HiPeRTA/-/tags/v4.0.0-dev)\n     - `HiPeRTA` uses `hdf5-R0` files converted from `simtel` by using `ctapipe_io_mchdf5` [v0.3.1-dev](https://github.com/cta-observatory/ctapipe_io_mchdf5/releases/tag/v0.3.1-dev)",
```
Comment (LukasNickel) [inline codemeta.json:15]: Why 0.6.3? We have 0.7.3 in the environment

Comment (garciagenrique) [reply codemeta.json:15]: releaseNote should be updated in the moment we'd do another release, and thus update with what everything is working

**Comment by garciagenrique** [comment]
New to me - do we all agree with these definitions ? For the record I found it for R contributions, and not sure how do they apply to Python ones, 

(Author) Use for full authors who have made substantial contributions to the package and should show up in the package citation.
(Contributor) Use for authors who have made smaller contributions (such as code patches etc.) but should not show up in the package citation.

File: codemeta.json (hunk 1 above)
Comment (LukasNickel) [reply codemeta.json:15]: Ah, there hasnt been a release since, ok

**Comment by LukasNickel** [comment]
> New to me - do we all agree with these definitions ? For the record I found it for R contributions, and not sure how do they apply to Python ones,
> 
> (Author) Use for full authors who have made substantial contributions to the package and should show up in the package citation. (Contributor) Use for authors who have made smaller contributions (such as code patches etc.) but should not show up in the package citation.

Haven't seen it so far aswell.

I suppose with that definition I might be a contributor? Up to you to decide

**Review by garciagenrique — APPROVED** [review]

**Pull request merged**